components/Navbar.js

```diff
@@ -1,45 +1,50 @@
 import Link from "next/link";
 import styled from "styled-components";
+import { Text, View, StyleSheet, TouchableOpacity } from "react-native-web";
 
-const Nav = styled.div`
-  border-top: 2px solid black;
-  border-bottom: 2px solid black;
-  display: flex;
-  background: #222;
-  padding: 0 15px;
-  list-style-type: none;
-  height: 55px;
-
-  ul {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
-  }
-
-  li {
-    display: inline;
-  }
-
-  img{
-
-  }
+const Image = styled.img.attrs(({ url }) => ({
+  src: url,
+}))`
```

Comment:
what is that tick (`) at the end ?

Reply:
Backticks to start using styled component notation/css